### PR TITLE
ZKP Key Consistency Check

### PR DESCRIPTION
**Pull request by stechu**
- add a test to check whether the ZKP keys generated is consistent with the ones defined at `manta-crypto`
- bump version

### CODE REVIEW

**Comment by stechu** [comment]
We should expect the test fail since currently the key is inconsistent. This need to be merged after manta-crypto's zkp keys being fixed.

**Review by bhgomes — APPROVED** [review]
👍

**Comment by bhgomes** [comment]
- @stechu My side of the review is done, you can also review the code I just added.
- @stechu @Kevingislason I changed some of the features so that we don't need to have `std` enabled to run key generation, since it's not required. Only file-IO requires `std`.

File: src/zkp/keys.rs [hunk 1]
```diff
@@ -14,239 +14,276 @@
...
+pub const COMMITMENT_PARAMS_SEED: [u8; 32] = [2; 32];
+
+/// Key Derivation Seed
+pub const KEY_DERIVATION_SEED: [u8; 32] = [3; 32];
+
+/// Pseudorandom Number Generator Salt
+pub const RNG_SALT: [u8; 32] = [
+	0x74, 0x68, 0x69, 0x73, 0x20, 0x69, 0x73, 0x20, 0x61, 0x20, 0x73, 0x65, 0x65, 0x64, 0x20, 0x66,
+	0x6f, 0x72, 0x20, 0x6d, 0x61, 0x6e, 0x74, 0x61, 0x20, 0x7a, 0x6b, 0x20, 0x74, 0x65, 0x73, 0x74,
+];
```
Comment (stechu) [inline src/zkp/keys.rs:54]: 👍

**Review by stechu — COMMENTED** [review]
LGTM. Can you add the fixed parameter test to the change log?

**Pull request merged**